tests/utils/inputparams/testparams.go

```diff
@@ -53,6 +54,15 @@ func GetContainerNameWithTimeStamp(containerName string) string {
 	return containerName + "_container_" + strconv.FormatInt(time.Now().Unix(), 10)
 }
 
+func GetVolumeNameOfSize(size int) string {
+	const chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
While eyeballing the test execution it is lot easier to know how far test execution has reached and what is going on now? on the other hand it is just a personal opinion, a lot more convenient to me; as we go along test name is our goto keyword like "refcount volume", "cloneVol" that helps to quickly know about the test it belongs. It is pretty handy to have referenced test name.

We may treat some special case as exception like L58.

We can use `C.TestName()` that gocheck is providing to get the current testcase name to keep it consistent.